### PR TITLE
RSA SigVer Capability Registration & Multiple RSA Cap Modes Registration

### PR DESCRIPTION
**Pull request by rygrange**
Added capability registration for RSA SigVer, along with support for registering multiple signature type capabilities for both SigVer and SigGen. *Multiple KeyGen capability registration was not added*

### CODE REVIEW

File: app/app_main.c [hunk 1]
```diff
@@ -359,69 +359,134 @@ int main(int argc, char **argv)
...
+	    rv = acvp_enable_rsa_cap_sig_type_parm(ctx, ACVP_RSA, ACVP_RSA_MODE_SIGVER, RSA_SIG_TYPE_PKCS1V15, MOD_RSA_4096, ACVP_RSA_SHA_512);
+	    CHECK_ENABLE_CAP_RV(rv);
+	// RSA w/ sigType: X9.31
+	  rv = acvp_enable_rsa_cap_parm(ctx, ACVP_RSA, ACVP_RSA_MODE_SIGVER, ACVP_SIG_TYPE, RSA_SIG_TYPE_X931);
+	  CHECK_ENABLE_CAP_RV(rv);
+//	  rv = acvp_enable_rsa_cap_sig_type_parm(ctx, ACVP_RSA, ACVP_RSA_MODE_SIGVER, RSA_SIG_TYPE_X931, MOD_RSA_2048, ACVP_RSA_SHA_224);
+//	  CHECK_ENABLE_CAP_RV(rv);
+	  rv = acvp_enable_rsa_cap_sig_type_parm(ctx, ACVP_RSA, ACVP_RSA_MODE_SIGVER, RSA_SIG_TYPE_X931, MOD_RSA_2048, ACVP_RSA_SHA_256);
+	  CHECK_ENABLE_CAP_RV(rv);
+//	  rv = acvp_enable_rsa_cap_sig_type_parm(ctx, ACVP_RSA, ACVP_RSA_MODE_SIGVER, RSA_SIG_TYPE_X931, MOD_RSA_2048, ACVP_RSA_SHA_512);
```
Comment (ashman-p) [inline app/app_main.c:453]: A general comment ... you may want to remove the commented lines of code.

**Review by bfussell — CHANGES_REQUESTED** [review]
I would omit the acvp_app binary from the changeset.
Otherwise looks good !

**Comment by rygrange** [comment]
On the acvp_app binary should I just ignore it on my commit or do we want to add it to the git ignore file? @bfussell

**Comment by bfussell** [comment]
Just ignore it on the commit

**Review by bfussell — APPROVED** [review]

**Comment by rygrange** [comment]
Making a new pull request with the suggestions that Norm and Barry asked for.

**Pull request closed without merging**